### ISSUE TITLE
Build a transfer learning model

### ISSUE THREAD

**Issue by DerikVo** [DerikVo/DSI_project_4_plant_disease#12]
Currently we have a model neural net of about 98% accuracy and F1 score on predicting if a tomato leaf has blight or is healthy. The next step is to build a transfer learning model to determine if a leaf has early blight or late blight. Then we will evaluate that model, as well as using prebuilt models.

**Comment by DerikVo** [comment]
Veda can you use [yasser's model](https://github.com/DerikVo/DSI_project_4_plant_disease/blob/yasser/notebooks/PlantVillageModeling_Yasser.ipynb) or a per-trained model from [keras](https://keras.io/api/applications/) to see if we can lower the number of false negatives?

We want to compare our model with a per-trained model to see how good it is.

**Comment by DerikVo** [comment]
You can also reference the Transfer learning (week-9/nn-transfer-learning) from breakfast hour

**Comment by DerikVo** [comment]
Never mind dont think this is a good use of our limited time since we already have a well performing model
